test/jdk/java/foreign/TestSegments.java

```diff
@@ -40,12 +42,9 @@
 import java.util.Spliterator;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
-import java.util.function.Consumer;
 import java.util.function.LongFunction;
 import java.util.stream.Stream;
 
-import org.testng.annotations.*;
```

Comment:
These import changes were the only real change in the merge - everything else was automatically merged